package.json

```diff
@@ -96,6 +96,7 @@
     "react": "^15.3.1",
     "react-addons-transition-group": "^15.3.0",
     "react-addons-update": "^15.3.1",
+    "react-copy-to-clipboard": "^5.0.0",
```

Comment:
We definitely don't **need** an NPM package, so yeah if @arantzardzm is up for fixing it up the more nitty gritty way go ahead, I don't think I'm way too scared of the extra package there though.

I believe what the package probably handles is also legacy cases where the new API is not supported and stuff that might be a bit of a pain.